app/src/main/java/com/cornellappdev/scoop/ui/components/search/FirstPage.kt

```diff
@@ -0,0 +1,196 @@
+                            contentDescription = stringResource(R.string.departure_location),
+                            modifier = Modifier
+                                .padding(end = 10.dp)
+                                .size(36.dp)
+                                .align(Alignment.CenterVertically)
+                        )
+                        CityPicker(
+                            cityState = departureLocation,
+                            placeholder = stringResource(R.string.departure_location),
+                            placeholderColor = PlaceholderDarkGray,
+                            modifier = Modifier.align(Alignment.Bottom)
+                        ) {
+                            searchState.value = searchState.value.copy(departureLocation = it)
+                        }
+                    }
+                    Row(
+                        modifier = Modifier
+                            .padding(top = 25.dp)
+                            .align(Alignment.CenterHorizontally)
+                    ) {
+                        Icon(
+                            imageVector = Icons.Outlined.Place,
+                            contentDescription = stringResource(R.string.arrival_location),
+                            modifier = Modifier
+                                .padding(end = 10.dp)
+                                .size(36.dp)
+                                .align(Alignment.CenterVertically)
+                        )
+                        CityPicker(
+                            cityState = departureLocation,
+                            placeholder = stringResource(R.string.arrival_location),
+                            placeholderColor = PlaceholderDarkGray,
+                            modifier = Modifier.align(Alignment.Bottom)
+                        ) {
+                            searchState.value = searchState.value.copy(departureLocation = it)
+                        }
+                    }
+                    Row(
+                        modifier = Modifier
+                            .padding(top = 25.dp)
+                            .align(Alignment.CenterHorizontally)
+                    ) {
+                        Icon(
+                            imageVector = Icons.Outlined.CalendarToday,
+                            contentDescription = stringResource(R.string.calendar_icon_descrption),
+                            modifier = Modifier
+                                .padding(end = 10.dp)
+                                .size(36.dp)
+                                .align(Alignment.CenterVertically)
+                        )
+                        TextButton(
+                            onClick = { /*TODO*/ },
+                            contentPadding = PaddingValues(0.dp),
+                        ) {
+                            Column {
+                                Text(
+                                    text = stringResource(R.string.departure_date),
```

Comment:
The text should only be the placeholder given that the departureDate state is empty, similarly for the color of the text as well!